frontend/src/api/User.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import {BASE_URL} from "./Commons";
+
+const URL = BASE_URL + "/auth";
+
+export const loginUser = async (request) => {
+  const fetchData = await axios.post(URL + "/login", request);
+  return fetchData.data;
+};
+
+export const getCurrentUser = async (token) => {
+  return axios.get(URL + "/getuser", {headers: { Authorization: "Bearer " + token}});
+};
```

Comment:
You need to figure out a way to always send the token after it's been set. The token needs to be included with **every** request (login, register, etc.) and then backend decides which endpoints are protected and which are not. For example, this function would take no arguments and include the token if the user is logged in. The function above, `loginUser` would do the same and so on.